scripts/parse.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-LOC_LIST=( "mensa-arcisstr" "mensa-garching" "mensa-leopoldstr" "mensa-lothstr" \
+CANTEEN_LIST=( "mensa-arcisstr" "mensa-garching" "mensa-leopoldstr" "mensa-lothstr" \
```

Comment:
Can we replace this list, to read the available canteens from the python enum?

Reply:
I like your idea. But I think using the shell script to parse the enums to a list and then call `main.py` for each canteen would not be the proper way. Instead, we should add an option directly to `main.py` to parse all canteens at once. I have created an issue for that. See #70.